ci/code_checks.sh

```diff
@@ -184,6 +184,11 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -R --include="*.rst" ".. ipython ::" doc/source
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    # Check for blank lines after the class definition
```

Comment:
I don't think we need this comment, it's already explained in the message in the next line.